pkg/gardenadm/staticpod/translator.go

```diff
@@ -55,22 +55,26 @@ func translatePodTemplate(ctx context.Context, c client.Client, objectMeta metav
 
 	return append([]extensionsv1alpha1.File{{
 		Path:        filepath.Join(kubelet.FilePathKubernetesManifests, pod.Name+".yaml"),
-		Permissions: ptr.To[uint32](0600),
+		Permissions: ptr.To[uint32](0640),
```

Comment:
Is this really necessary or just for consistency? The manifest files are read by `kubelet`, which needs to run as `root` anyway.

Reply:
I think we did this for consistency reasons. But given that the pod manifests shouldn't contain anything critical, it's probably fine.